docs/functional-components/static-assets.md

```diff
@@ -0,0 +1,26 @@
+## Static assets
+
+## Recommended packages
+
+* [express-static](https://expressjs.com/en/starter/static-files.html). 
+Express static is part of the express.js package that allows developers to expose static middlewares.
+
+## Guidance
+
+For serving static resources we recomend using `express.static` middleware as it has been widely used and tested in production.
+When using express.static() method, we can server static resources directly by specifying the folder name where we have stored our static resources.
+
+Documentation for the middleware can be found in:
+https://expressjs.com/en/starter/static-files.html
+
+### Caching
+
+The static middleware does server-side caching. 
+It lets you do two methods of client-side caching: ETag and Max-Age
```

Comment:
Cover only max-age - we need to change default argument. 
Reference spec: http://expressjs.com/en/resources/middleware/serve-static.html